databricks_template_schema.json

```diff
@@ -132,17 +132,82 @@
       "type": "string",
       "description": "\nWhether to use the Model Registry with Unity Catalog",
       "default": "no",
-      "enum": ["yes", "no"],
+      "enum": ["yes", "no"]
+    },
+    "input_staging_catalog_name": {
+      "order": 12,
+      "type": "string",
+      "description": "\nName of the catalog in Unity Catalog that will host the staging UC resources. \nThis catalog must already exist and service principals must have access to it.\nDefault",
+      "default": "staging",
       "skip_prompt_if": {
-        "properties": {
-          "input_setup_cicd_and_project": {
-            "const": "CICD_Only"
+        "anyOf": [
+          {
+            "properties": {
+              "input_include_models_in_unity_catalog": {
+                "const": "no"
+              }
+            }
+          },
+          {
+            "properties": {
+              "input_setup_cicd_and_project": {
+                "const": "Project_Only"
+              }
+            }
           }
-        }
+        ]
+      }
+    },
+    "input_prod_catalog_name": {
+      "order": 13,
+      "type": "string",
+      "description": "\nName of the Prod Unity Catalog.\nThis catalog must already exist and service principals must have access to it.\nDefault",
```

Comment:
I meant to do this `Name of the catalog in Unity Catalog that will host the staging UC resources.` but replace staging with production